express/index.html

```diff
@@ -1,279 +1,428 @@
-			SysAdminas</a>
-		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
-			aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-			<span class="navbar-toggler-icon"></span>
-		</button>
-		<div class="collapse navbar-collapse" id="navbarNav">
-			<ul class="navbar-nav">
-				<li class="nav-item active">
-					<a class="nav-link" href="#about">Sobre nós <span class="sr-only">(current)</span></a>
-				</li>
-				<li class="nav-item">
-					<a class="nav-link" href="#initiatives">Iniciativas</a>
-				</li>
-				<li class="nav-item">
-					<a class="nav-link" href="#award">Reconhecimentos</a>
-				</li>
-				<li class="nav-item">
-					<a class="nav-link" href="#team">Nosso time</a>
-				</li>
-				<li class="nav-item">
-					<a class="nav-link" href="#">Contato</a>
-				</li>
-				<li class="nav-item">
-					<a class="nav-link" href="#">Eventos</a>
-				</li>
-			</ul>
-		</div>
-	</nav>
+    <!-- ======= Hero Section ======= -->
+    <section id="hero">
+      <div class="hero-container" data-aos="zoom-in" data-aos-delay="100">
+        <h1>
+          <span style="color: #ff66c4"
+            ><i class="fas fa-chevron-right"></i
+          ></span>
+          Sysadminas
+        </h1>
+        <div class="typewriter">
+          <h2>SysAdmin Like a Girl</h2>
+        </div>
+        <a href="#about" class="btn-get-started">Conhecer</a>
+      </div>
+    </section>
+    <!-- End Hero Section -->
 
-	<!-- ======= Hero Section ======= -->
-	<section id="hero">
-		<div class="hero-container" data-aos="zoom-in" data-aos-delay="100">
-			<h1><span style="color: #ff66c4"><i class="fas fa-chevron-right"></i></span> Sysadminas</h1>
-			<div class="typewriter">
-				<h2>SysAdmin Like a Girl</h2>
-			</div>
-			<a href="#about" class="btn-get-started">Conhecer</a>
-		</div>
-	</section><!-- End Hero Section -->
+    <main id="main">
+      <!-- ======= About Section ======= -->
+      <section id="about">
+        <div class="container" data-aos="fade-up">
+          <div class="row about-container">
+            <div class="col-lg-6 content order-lg-1 order-2">
+              <h2 class="title">Um pouco sobre nós</h2>
+              <p class="text-justify">
+                A comunidade <strong>SysAdminas</strong> é um grupo inclusivo
+                que fornece uma rede de suporte para as
+                <strong>mulheres</strong> que desejam aprender conteúdos
+                relacionados a <strong>infraestrutura</strong> no Brasil.
+              </p>
+              <p class="text-justify">
+                Nossas iniciativas são <strong>gratuitas</strong> e organizadas
+                pela <strong>comunidade</strong>.
+              </p>
+            </div>
+          </div>
+          <div class="row about-container">
+            <div class="col-lg-6 content order-lg-1 order-2">
+              <h4 class="custom-bold">Nossa história</h4>
```

Comment:
Can you fix that margin between "Nossas iniciativas são..." and "Nossa história"?

![image](https://user-images.githubusercontent.com/15144708/95003597-38800480-05b7-11eb-96f0-5d09f681ed61.png)

Reply:
Can you change indentation to tab? Because it may cause some conflicts for other contributors.